pkg/performanceprofile/controller/performanceprofile/components/utils.go

```diff
@@ -142,3 +152,11 @@ func CPUMaskToCPUSet(cpuMask string) (cpuset.CPUSet, error) {
 
 	return builder.Result(), nil
 }
+
+func ListToString(cpus []int) string {
```

Comment:
Function title is a bit general for int list to string.
For general knowledge , there are some one liners that could do the same such as:
`strings.Trim(strings.Join(strings.Fields(fmt.Sprint(int_slice)), ","), "[]")`

Reply:
I asked about avoiding a similar oneliner, and I still think we should avoid them in general

Reply:
note taken

Reply:
this was the original one
```
func stringSplit(cpuListSlice []int) string {
	return strings.Trim(strings.Join(strings.Fields(fmt.Sprint(cpuListSlice)), ","), "[]")
}
```
Changed due a comment request

Reply:
yes, comment was from me. Let's please keep it in the current form (and place?)

Reply:
I've mode to utils that make more sense for me, its an auxiliary func, right?